zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/util/ElasticsearchRepository.java

```diff
@@ -35,7 +36,9 @@ public class ElasticsearchRepository {
   protected final Logger logger;
 
   public ElasticsearchRepository(
-      final ElasticsearchAsyncClient client, final Executor executor, final Logger logger) {
+      @WillCloseWhenClosed final ElasticsearchAsyncClient client,
```

Comment:
🤡 What does this annotation? It looks wild.

Reply:
It doesn't "do" anything (unless you use some compiler add ons or google's error prone), but it's still helpful imo :)